### PR TITLE
geospatial v1 experimental support: featuretypes

### PR DESCRIPTION
**Pull request by vincent-cognite**
The business objects defined are the simplest form for now. This is to get started and we can improve when our clients give us feedback.

### CODE REVIEW

**Comment by pawelzwolenskicognite** [comment]
what about domain? it looks like SDK works always with default one

File: cognite/experimental/_api/geospatial.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                ... ]
+                >>> res = c.geospatial.create_feature_types(feature_types)
+
+
+        Args:
+
+        Returns:
+            FeatureType: The created feature types.
+        """
+        return self._create_multiple(items=feature_type)
```
Comment (pawelzwolenskicognite) [inline cognite/experimental/_api/geospatial.py:45]: I'm definitely not a Python guy... where is _create_multiple defined?

**Review by pawelzwolenskicognite — APPROVED** [review]
good start!

File: cognite/experimental/_api/geospatial.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        Returns:
+            Union[FeatureType, FeatureTypeList]: Created feature type definition(s)
+
+        Examples:
+
+            Create new type definitions::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> from cognite.experimental.data_classes import Type
```
Comment (tuanng-cognite) [inline cognite/experimental/_api/geospatial.py:31]: import FeatureType, right?

**Review by tuanng-cognite — APPROVED** [review]
Very nice!

File: cognite/experimental/_api/geospatial.py (hunk 2 above)
Comment (vincent-cognite) [reply cognite/experimental/_api/geospatial.py:31]: well spotted!

**Comment by vincent-cognite** [comment]
> what about domain? it looks like SDK works always with default one

I was wondering if this should be added now. I was assuming this will be used by APIs sitting on top (and written in Java or Kotlin). Did not want to confuse python user (well, maybe it does anyway)

We can retrofit later easily.

**Pull request merged**